java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/ObjectPrx.java

```diff
@@ -211,108 +210,108 @@ CompletableFuture<Object.Ice_invokeResult> ice_invokeAsync(
             Map<String, String> context);
 
     /**
-     * Returns the identity embedded in this proxy.
+     * Gets the identity embedded in this proxy.
      *
      * @return The identity of the target object.
      */
     Identity ice_getIdentity();
 
     /**
-     * Returns a proxy that is identical to this proxy, except for the identity.
+     * Creates a proxy that is identical to this proxy, except for the identity.
      *
      * @param newIdentity The identity for the new proxy.
-     * @return The proxy with the new identity.
+     * @return A proxy with the new identity.
      */
     ObjectPrx ice_identity(Identity newIdentity);
 
     /**
-     * Returns the per-proxy context for this proxy.
+     * Gets the per-proxy context for this proxy.
      *
      * @return The per-proxy context. If the proxy does not have a per-proxy (implicit) context, the
-     *     return value is <code>null</code>.
+     *     return value is null.
```

Comment:
It's interesting. Every other keyword wrapped in `<code>` tags was mapped to use the new `{@code }` tag instead... except for `null`. I'm not aware of any reason why `null` is less deserving of `{@code }` than `true`.
But all of the `null`s in this file had their code formatting stripped.